### PR TITLE
Adding Markdown Feature and Updating README file - Issue #6

### PR DESCRIPTION
**Pull request by GMOTGIT**
Hello Leyang,
I added to your code the function getMdToHtml, this function works exactly like the getHtml function, but parse the markdowns to <h123> tags. 

Also updated readFile function and getUserInput function to accept .md files too.

Also, updated the README file to include the markdown feature now.

Please, let me know if I need to change anything,
Thank you so much.

### CODE REVIEW

**Comment by GMOTGIT** [comment]
Also the required syntax is not working for me, so I had to go with import to make it work.

**Comment by lyu4321** [comment]
Hey Gustavo, thank you so much for your changes! There are just a few things I would like to mention:

- In the README, "## H3 Heading" should be ### instead of ##
- Would it be possible to add some examples of using ## and ### in the README example? 
- The getMdToHtml function is very similar to the getHtml function. Would it be possible to combine the two into one function to avoid repeating the code?

Thank you :)

File: README.md [hunk 1]
```diff
@@ -126,6 +126,72 @@ In the dist folder, the following files will be generated:<br/>
...
 
+# Additional support for md (markdown) files
+
+For markdown files (.md), the program will parse markdown to HTML tags.
+
+| MD File         | HTML File             | Webpage             |
+| --------------- | --------------------- | ------------------- |
+| `# H1 Heading`  | `<h1>H1 Heading</h1>` | <h1>H1 Heading</h1> |
+| `## H2 Heading` | `<h2>H2 Heading</h2>` | <h2>H2 Heading</h2> |
+| `## H3 Heading` | `<h3>H3 Heading</h3>` | <h2>H3Heading</h2>  |
```
Comment (lyu4321) [inline README.md:137]: Hello, there is a small typo here, it should be ### :)

**Review by lyu4321 — CHANGES_REQUESTED** [review]

**Comment by GMOTGIT** [comment]
Hello Leyang,
I fixed the ReadMe.md file, sorry for the typo on line 137.
I also merged getHtml and getMdToHtml.
In order to do that I had to introduce another parameter to getHtml function so now the signature is:
getHtml (file, isTxt).
I had to do that because I could not get the type of the 'file' parameter inside the function, as it came as a string from readFile function.
So I get the file type inside readFile and pass it to getHtml.

Let me know if you are happy with these changes.
Thank you so much.

**Comment by lyu4321** [comment]
Hi! Thanks so much for the fixes, I've merged the PR :)

**Pull request merged**